test/GenomePositionSearchBoxTest.js

```diff
@@ -298,5 +311,18 @@ describe('Genome position search box tests', () => {
     it("checks that the div hasn't grown too much", () => {
       expect(div.clientHeight).toBeLessThan(500);
     });
+
+    it('Cleans up', () => {
```

Comment:
This test was missing a cleanup method but I only just noticed after reordering the tests in karma.conf.js.a